### DISCUSSION TITLE
custom pagination

### DISCUSSION THREAD

**Discussion by rpimentaf** [ChilliCream/hotchocolate#5394] [Q&A]
Hi, everybody!!! I've customized the pagination of a resolver and it now return an object that is not a simple connection object. So, i'm trying to return a custom connection object from the resolver but the following code does not compile in Visual Studio...
```
var connection = new Connection<User>(
    edges,
    pageInfo,
    getTotalCount: cancellationToken => ValueTask.FromResult(0));
```
The error that is shown says that FromResult is not a method of ValueTask. Is it correct? If so,  I do not understand what i have to edit... Any help? Tks!!! (editado)

**Comment by tobias-tengler** [comment]
Are you on >= .NET 5? It was introduced in .NET 5.

**Reply by rpimentaf** [reply]
Hi, Tobias! I'm using .Net Standard 2.1.... Is this the problem?

**Reply by cajuncoding** [reply]
Yes, that static method doesn't exist, instead just can just new() it up.... `new ValueTask(0)`....

MS Docs:
https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.valuetask-1.-ctor?view=netcore-2.1#system-threading-tasks-valuetask-1-ctor(-0)
